crates/rpc/rpc/src/trace.rs

```diff
@@ -257,7 +257,10 @@ where
 
         // ensure that the range is not too large, since we need to fetch all blocks in the range
         let distance = end.saturating_sub(start);
-        if distance > 100 {
+        let eth_config = EthConfig::default();
+        let max_blocks_per_filter = eth_config.max_trace_filter_blocks;
```

Comment:
we can't use `default()` here, we need to actually pass down the config into the trace api

see how we do it similarly in `EthFilterInner` here
https://github.com/paradigmxyz/reth/blob/57bb9ba6fbb148b4ce85918fa32f58fecc6538d4/crates/rpc/rpc/src/eth/filter.rs#L370-L371

and make it accessible in the rpc method here
https://github.com/paradigmxyz/reth/blob/57bb9ba6fbb148b4ce85918fa32f58fecc6538d4/crates/rpc/rpc/src/eth/filter.rs#L495